internal/storage/imapsql/imapsql.go

```diff
@@ -478,12 +478,12 @@ func (store *Storage) GetOrCreateIMAPAcct(username string) (backend.User, error)
 func (store *Storage) Lookup(key string) (string, bool, error) {
 	accountName, err := prepareUsername(key)
 	if err != nil {
-		return "", false, nil
+		return "", false, err
```

Comment:
I believe it should stay `nil` for the same reason as other Lookup implementations you changed.

Reply:
Got it, done.